### PR TITLE
[861] Auth by SSO-assigned roles

### PR DESCRIPTION
**Pull request by stephenhillier**
Added 4 roles: gwells_adjudicator, gwells_statutory_authority, gwells_viewer, and gwells_admin.

Users are added to the appropriate group automatically based on their Keycloak realm roles. Each group is created automatically when the first user with the corresponding role logs on.

Adding and editing of models is protected by Django's out of the box permission system. Default permissions are assigned to each of the groups in `gwells/permissions.py` but they can also be changed in the Django admin panel.

### CODE REVIEW

**Comment by stephenhillier** [comment]
[EPIC#Login | Roles and Responsibilities (Django user roles) for Registry V2 |3| !!!](https://trello.com/c/N0abLj79/861-epiclogin-roles-and-responsibilities-django-user-roles-for-registry-v2-3)

**Comment by stephenhillier** [comment]
[EPIC#Registry | Issues for fixing |3| !!!](https://trello.com/c/jKdRNaSv/960-epicregistry-issues-for-fixing-3)

File: app/frontend/src/registry/components/people/OrganizationAdd.vue [hunk 1]
```diff
@@ -253,6 +257,9 @@ export default {
         website_url: [],
         email: []
       }
+    },
+    formatTel (value) {
```
Comment (Sybrand) [inline app/frontend/src/registry/components/people/OrganizationAdd.vue]: Can we move this into a common area somewhere? It's copied into 4 different places.

**Comment by stephenhillier** [comment]
[EPIC#Registry | Issues for fixing |2| !!!](https://trello.com/c/rMOkDRc2/977-epicregistry-issues-for-fixing-2)

**Comment by stephenhillier** [comment]
[EPIC#Registry | Issues for fixing |2| !!!](https://trello.com/c/rMOkDRc2/977-epicregistry-issues-for-fixing-2)

**Pull request merged**